test/functional/api/v2/locations_controller_test.rb

```diff
@@ -241,4 +241,30 @@ def add_locations
 
     assert_equal expected_metadata, response.except('results')
   end
+
+  context "organization controller tests" do
```

Comment:
Sorry, I didn't mean to move the organisation controller tests in here directly - but simply change them to use the location controller and remain here.  They're the same thing.